examples/contest_result.xml

```diff
@@ -3,4 +3,5 @@
   <jurisdiction_id>20013</jurisdiction_id>
   <entire_district>no</entire_district>
   <total_votes>100</total_votes>
+  <certification>unofficial_complete</certification>
```

Comment:
`certification` isn't an element. It's just an attribute. You can just remove this here since the attribute exists (i.e. `certification="certified"`).